src/underground/models.py

```diff
@@ -211,35 +211,47 @@ def get(route_or_url: str, retries: int = 100, api_key: str = None) -> "SubwayFe
             )
         )
 
-    def extract_stop_dict(self, timezone: str = metadata.DEFAULT_TIMEZONE) -> dict:
+    def extract_stop_dict(
```

Comment:
this is great!  do you think this arg should be exposed to the cli as well? by adding an arg here https://github.com/nolanbconaway/underground/blob/master/src/underground/cli/stops.py#L48

Reply:
Done!